# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../devilspie/remove-WNCK_WINDOW_MODAL_DIALOG.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


 WNCK_WINDOW_MODAL_DIALOG doesn't exist since libwnck 2.19.4

diff -ruN devilspie-0.21.orig/src/xutils.c devilspie-0.21/src/xutils.c
--- devilspie-0.21.orig/src/xutils.c	2007-01-05 22:08:40.000000000 +0100
+++ devilspie-0.21/src/xutils.c	2007-10-06 11:21:34.000000000 +0200
@@ -305,9 +305,6 @@
   case WNCK_WINDOW_DIALOG:
     atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_DIALOG");
     break;
-  case WNCK_WINDOW_MODAL_DIALOG:
-    atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_MODAL_DIALOG");
-    break;
   case WNCK_WINDOW_TOOLBAR:
     atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_TOOLBAR");
     break;
